tests/test_consumer.py

```diff
@@ -2,16 +2,18 @@
 # See LICENSE file for licensing details.
 
 import json
+import re
 import unittest
 from unittest.mock import patch
 
 from ops.charm import CharmBase
 from ops.framework import StoredState
 
-# from ops.model import Network
 from ops.testing import Harness
+import charms.prometheus_k8s.v0.prometheus as prometheus_lib
```

Comment:
This is one reason why I prefer "static" class variables, such as `PrometheusConsumer.ALERT_RULES_PATH`. Just saying :D

Reply:
This was a good suggestion. Adopted now. Thanks.